src/routes/assistants/+page.server.ts

```diff
@@ -14,12 +14,21 @@ export const load = async ({ url }) => {
 
 	const modelId = url.searchParams.get("modelId");
 	const pageIndex = parseInt(url.searchParams.get("p") ?? "0");
+	const createdByName = url.searchParams.get("user");
+
+	if (createdByName) {
+		const existingUser = await collections.users.findOne({ hfUserId: createdByName });
+		if (!existingUser) {
+			throw error(404, `User "${createdByName}" doesn't exist`);
+		}
+	}
 
 	// fetch the top assistants sorted by user count from biggest to smallest, filter out all assistants with only 1 users. filter by model too if modelId is provided
 	const filter: Filter<Assistant> = {
 		userCount: { $gt: 1 },
 		modelId: modelId ?? { $exists: true },
 		featured: true,
+		createdByName: createdByName ?? undefined,
```

Comment:
it's doesn't seems to work.

Reply:
https://github.com/huggingface/chat-ui/pull/773/commits/13dfcc34424804ccc7365e6a1ebe524464bac725